src/templates/article.js

```diff
@@ -53,13 +54,14 @@ const ArticleTemplate = props => {
   const convertkitURL = props.data.site.siteMetadata.convertkit.url
 
   const pathname = post.fields.slug
+
   return (
     <Layout location={props.location} title={siteTitle}>
       <SEO
         description={post.frontmatter.description || post.excerpt}
         pathname={pathname}
         title={post.frontmatter.title}
-        image={post.frontmatter.image}
+        image={post.frontmatter.image.publicURL}
```

Comment:
Is this auto generated or will I have to manually specify it everywhere?

Reply:
autogenerated